utilities/logicApp/Callback-Workflow/workflow.json

```diff
@@ -344,7 +344,14 @@
                 "inputs": {
                     "schema": {
                         "type": "object",
-                        "properties": {}
+                        "properties": {
+                            "id": {
+                                "type": "string"
+                            },
+                            "state": {
+                                "type": "string"
+                            }
```

Comment:
Please remove as these properties are not included in the body. This only validates the body.